pyproject.toml

```diff
@@ -62,7 +62,7 @@ vp-deltachi2 = "validphys.scripts.vp_deltachi2:main"
 # Generic dependencies (i.e., validphys)
 python = "^3.9"
 matplotlib = ">=3.3.0,<3.8"
-pineappl = "^0.7.0"
+pineappl = "^0.8.2"
```

Comment:
I'm not a friend of setting very strict requirements, `0.8.0` should work as well. Given 0-1-2 were released almost together it's ok

(also, I hope going forward, there's not many more changes of api, having to change code just because channels is more correct than lumi is a bit annoying...)

Reply:
> I'm not a friend of setting very strict requirements, 0.8.0 should work as well.

Likewise! Although I'd keep it this way because for whatever reason `pineko` (given that `pineko` now depends on `nnpdf`) would pick versions between  `<=v0.8,0<v0.8.2` which will not work.

> (also, I hope going forward, there's not many more changes of api, having to change code just because channels is more correct than lumi is a bit annoying...)

I have also always advocated for this...

Reply:
> given that pineko now depends on nnpdf

The dependency of pineko in nnpdf is optional. But yes, the problem with being very restrictive and changing the api without necessity (in _this_ case I guess there was necessity) is that the different projects will start clashing with each other.